Content.Server/Speech/EntitySystems/ReplacementAccentSystem.cs

```diff
@@ -62,6 +62,7 @@ public string ApplyReplacements(string message, string accent)
                 // this is kind of slow but its not that bad
                 // essentially: go over all matches, try to match capitalization where possible, then replace
                 // rather than using regex.replace
+#pragma warning disable RA0026 // Use of uncached static Regex function
```

Comment:
This warning is valid, suppressing it is like this will cause it to go unnoticed rather than getting it fixed properly.

This code is *really bad* and likely causes tons of unnecessary regex parsings, they need to be cached.